import_hior.py

```diff
@@ -177,10 +178,14 @@ def save_to_csv(dataset: list[dict], filename: str):
 
 
 def upload_data_to_blob_storage(datasets):
-    # Get connection string and container name from environment variables
-    connection_string = os.environ.get('AZURE_STORAGE_CONNECTION_STRING')
+    # Get storage-url and container name from environment variables
+    account_url = os.environ.get("AZURE_STORAGE_CONTAINER")
```

Comment:
Moeten we dit niet iets van container_name noemen?

Reply:
dit is de een env variabele met de storage url die ook door de frontend wordt gebruikt